### DISCUSSION TITLE
Directivity Score

### DISCUSSION THREAD

**Discussion by coda101** [pierreaubert/spinorama#287] [Ideas]
I was interested in how a directivity score might be calculated. I feel it is a very important quality indicator.

**Comment by pierreaubert** [comment]
I have been thinking about it. Something around how large is the
deviation from linear DI  between 500hz and 10kHz or something like that.
What is difficult is how do you correlate whatever computation with
preferences?

Influence of DI is likely to be highly correlated with your room.

Ideas?

On Tue, Aug 6, 2024 at 9:05 PM coda101 ***@***.***> wrote:

> I was interested in how a directivity score might be calculated. I feel it
> is a very important quality indicator.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pierreaubert/spinorama/discussions/287>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AANBKH6NMK3JTVFSEIHZMW3ZQENBFAVCNFSM6AAAAABMC5ZAU6VHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGAYTSMZQGU>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
--
PfA

**Reply by coda101** [reply]
I was thinking too in terms of a flatness score over a given frequency range, perhaps weighted too. I've only read a couple of opinions regarding the correlation with preference; for example, Sean's talk with Erin: https://youtu.be/IEtYH03pfOI?t=3790
And you can also find some preference ratings against objective data from page 47 of Sean's presentation here: https://www.dropbox.com/s/25md4nta14dqity/Predicting%20Loudspeaker%20Sound%20Quality%20Erin%27s%20Audio%20Corner.pdf?dl=0

Few studies have hinted at a trend, though I do think it is reasonable to consider a smooth Directivity Index transition based on these preference ratings.

**Comment by pierreaubert** [comment]
The other parameter is the slope. you can be very flat but with a high
slope (very directional) or flattish (almost omni). They look orthogonal to
me.

On Wed, Aug 7, 2024 at 9:07 AM coda101 ***@***.***> wrote:

> I was thinking too in terms of a flatness score over a given frequency
> range, perhaps weighted too. I've only read a couple of opinions regarding
> the correlation with preference; for example, Sean's talk with Erin:
> https://youtu.be/IEtYH03pfOI?t=3790
> And you can also find some preference ratings against objective data from
> page 47 of Sean's presentation here:
> https://www.dropbox.com/s/25md4nta14dqity/Predicting%20Loudspeaker%20Sound%20Quality%20Erin%27s%20Audio%20Corner.pdf?dl=0
>
> Few studies have hinted at a trend, though I do think it is reasonable to
> consider a smooth Directivity Index transition based on these preference
> ratings.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pierreaubert/spinorama/discussions/287#discussioncomment-10261172>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AANBKH75FKRTJNGQSRLO6ETZQHBUTAVCNFSM6AAAAABMC5ZAU6VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMRWGEYTOMQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
--
PfA

**Reply by coda101** [reply]
That seems dependant on the room setup from what I've read in terms of preference. I can see the flatness score working with the decimal system, but maybe the slope could be represented in an unranked or impartial fashion. 

Maybe take an average of a few loudspeakers that have done well in subjective blind tests as a benchmark for a "middle ground" slope. 
Though, beyond the idea, I feel I might be in over my head. So it will be interesting to see what you come up with Pierre.

**Comment by pierreaubert** [comment]
Will work on it in #289
